airbyte-webapp/src/packages/cloud/views/users/InviteUsersHint/InviteUsersHint.test.tsx

```diff
@@ -0,0 +1,83 @@
+import { fireEvent, render } from "@testing-library/react";
+import React from "react";
+import { TestWrapper } from "test-utils/testutils";
+
+import { useExperiment } from "hooks/services/Experiment";
+import { Experiments } from "hooks/services/Experiment/experiments";
+import { RoutePaths } from "pages/routePaths";
+
+import { CloudSettingsRoutes } from "../../settings/routePaths";
+
+const mockToggleInviteUsersModalOpen = jest.fn<ReturnType<typeof useExperiment>, Parameters<typeof useExperiment>>();
```

Comment:
The typing of this mock seems wrong, maybe copy-pasted from `mockUserExperiment`?